modules/commons/src/main/frontend/src/components/ErrorDialog.jsx

```diff
@@ -61,12 +61,19 @@ const useStyles = makeStyles(theme => ({
 // </ErrorDialog>
 //
 const ErrorDialog = (props) => {
-  const { title, children, onClose, ...rest } = props;
+  const {
+    title = "Error",
+    children,
+    onClose,
+    maxWidth = "xs",
+    fullWidth = true,
+    ...rest
+  } = props;
 
   const classes = useStyles();
 
   return (
-    <Dialog onClose={onClose} {...rest}>
+    <Dialog onClose={onClose} title={title} maxWidth={maxWidth} fullWidth={fullWidth} {...rest}>
```

Comment:
`Dialog` doesn't have a `title` prop
```suggestion
    <Dialog onClose={onClose} maxWidth={maxWidth} fullWidth={fullWidth} {...rest}>
```